src/sentry/utils/snuba.py

```diff
@@ -726,6 +727,8 @@ def __init__(
         is_grouprelease=False,
         **kwargs
     ):
+        # TODO: instead of having events be the default, make dataset required.
```

Comment:
Tried doing this but it wasn't as straight forward as I thought it would be.

So, making `events` the default dataset for now while we try to expedite getting outcomes out. And then will come back and address this TODO.